death-causes-app/src/components/UnitPicker.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { OverlayTrigger } from "react-bootstrap";
+import { Tooltip } from "react-bootstrap";
+import { Button } from "react-bootstrap";
+
+export interface UnitPickerProps {
+  options: string[];
+  onChoice: (newUnitName:string) => void; //satser på at det går.
+  size: string;
+}
+
+class UnitPicker extends React.PureComponent<UnitPickerProps> {
+  chosenValue: string;
+  constructor(props: UnitPickerProps) {
+    super(props);
+    this.chosenValue=this.props.options[0];
+    this.handleToggleUnit=this.handleToggleUnit.bind(this);
+  }
+
+  handleToggleUnit(){
+    const i = this.props.options.indexOf(this.chosenValue)
+    if(i===this.props.options.length-1){
+      this.chosenValue=this.props.options[0]
+    }
+    else{
+      this.chosenValue=this.props.options[i+1]
+    }
+    this.props.onChoice(this.chosenValue);
+  }
+
+  renderToolTips(injected:any):React.ReactNode{
+    return( <Tooltip id="button-tooltip" {...injected}>
```

Comment:
hvad er `injected` her?

Reply:
injected er nogle argumenter som OverlayTrigger giver videre til sit overlay (for at placere overlay'et det rigtige sted).